manifests/globals.pp

```diff
@@ -70,6 +71,12 @@
         /^5\./ => '8.1',
         default => undef,
       },
+      'RedHat' => $::operatingsystemrelease ? {
+        /^7\./ => '9.2',
+        /^6\./ => '8.4',
+        /^5\./ => '8.1',
+        default => undef,
+      },
```

Comment:
This still looks wrong. Now it's just duplicating the `default` statement. I don't understand what's happening here.